Examples/Swift/ViewController.swift

```diff
@@ -298,21 +302,32 @@ class ViewController: UIViewController, MGLMapViewDelegate, CLLocationManagerDel
     }
     
     func setupMapView() {
-        guard mapView == nil else { return }
-        mapView = NavigationMapView(frame: view.bounds)
+        guard self.mapView == nil else { return }
+        let mapView = NavigationMapView(frame: view.bounds)
+        self.mapView = mapView
```

Comment:
maybe if we're explicitly dictating `self.mapView` and `mapView` we should probably use a different approach here right? 
Like so:
```
guard mapView == nil else { return }
mapView = NavigationMapView(frame: view.bounds)
```

Reply:
There's actually a reason for this pattern -- `self.mapView` is an optional, but since we can guarantee it's existence (because we created it), we use the local reference. It's cleaner than using force-unwraps.